### PR TITLE
Add simple script for plain startup of services without systemd

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: script/start-openqa [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+start_service() {
+    sudo -u geekotest $(dirname $0)/$1 daemon
```
Comment (Martchus) [inline script/start-openqa]: `$(dirname $0)/$1` lacks quotes.

**Comment by coolo** [comment]
I think this deserves some motivation and/or use cases

**Comment by Martchus** [comment]
The use-case is obviously local development (although it would make sense to state that explicitly e.g. in a comment on top of the script).

I prefer starting openQA as my regular user using `OPENQA_BASEDIR` instead of mixing the packaged/production setup with my development setup. My own helper scripts are also helping one to start multiple instances in parallel and to start workers, gru and other daemons. But for very basic web UI development this script would actually make sense as it minimizes the setup effort, e.g. there's no extra database (owned by the regular user) required.

**Comment by okurz** [comment]
Yeah sorry, was distracted and wanted to write about about this idea. So yes, it's for local development, hence I delete it in the install make target. Same as Martchus I have some local scripts which I didn't upstream yet and wanted to see if we can find a better place for them, now it's the turn for the local startup script. It unfortunately again *slightly* duplicates what we already have in other scripts already. The problem with systemd services is for me that I would not know how to use the systemd files intended for system-wide installation with my local checkout openQA instance. Other opinions?

**Comment by kraih** [comment]
I don't find the script particularly helpful, it doesn't even cover starting all services.

**Comment by okurz** [comment]
like which?

**Comment by kraih** [comment]
Looks like @Martchus already mentioned it above, all the other daemons we have, like worker, gru and so on. Personally, i'm also using @Martchus 's [openQA-helper scripts](https://github.com/Martchus/openQA-helper), they have the advantage that they cover the whole stack (including os-autoinst development).

I'm not opposed to the idea of including such scripts in openQA itself, but they should come with a clearly defined workflow for local development imo. and cover the full stack (tests, os-autoinst and openQA).

**Comment by okurz** [comment]
@Martchus maybe you can propose some startup helper from your repos for inclusion instead?

**Comment by Martchus** [comment]
> but they should come with a clearly defined workflow

That's true. So far I didn't want to spoil the "generality" of the main openQA repository with my scripts which are focusing on a certain workflow.

> maybe you can propose some startup helper from your repos for inclusion instead?

So far I have already upstreamed documentation from my helpers. I could start upstreaming the actual helpers as well (preferably putting them into their own sub directory). However, my repository does *not* contain a fully automatic setup script (and the pieces which are automated are not regularly tested). That means at least that the manual setup instructions needed to be moved as well.

So maybe I'll just continue with moving further documentation to the main openQA documentation and add a reference to my repo within the setup guide.

**Comment by kraih** [comment]
Yea, setup was the one problem i still had with @Martchus 's scripts. Would be nice if that could be automated somehow.

**Comment by Martchus** [comment]
Of course the idea was that one knows in the end what's going on but for instance handling the API key/secret could be simplified. Known problems should be fixed by now but there might still be things not working (e.g. need to be adapted to upstream changes).

**Pull request closed without merging**